includes/class-omise-capabilities.php

```diff
@@ -68,6 +72,27 @@ public static function retrieve($pKey = null, $sKey = null)
 		return self::$instance;
 	}
 
+	/**
+	 * @return boolean
+	 */
+    public static function shouldCallApi() {
```

Comment:
@ajzkk should we match the existing indentation of the function `retrieve()` above?